webapp/src/components/backstage/playbooks/playbook.tsx

```diff
@@ -120,9 +121,16 @@ const Playbook = () => {
                             <SubTitle>{subTitle}</SubTitle>
                         </HorizontalBlock>
                     </VerticalBlock>
+                    {
+                        archived &&
+                        <StatusBadge
+                            data-testid='archived-badge'
+                            status='Archived'
+                        />
```

Comment:
Doesn't the linter enforce the curly braces here? I thought that was needed

```suggestion
                        <StatusBadge
                            data-testid={'archived-badge'}
                            status={'Archived'}
                        />
```